app/src/main/java/com/educacionit/biciya/models/response/ResponseDataClass.kt

```diff
@@ -0,0 +1,26 @@
+package com.educacionit.biciya.models.response
+
+import com.google.gson.annotations.SerializedName
+
+data class StationInformationResponse(
+    val last_updated: Long,
+    val ttl: Int,
+    val data: StationData
+)
+
+data class StationData(
+    val stations: List<Station>
+)
+
+data class Station(
+    val station_id: String,
+    val name: String,
+    val lat: Double,
+    val lon: Double,
+    val address: String?,
+    val cross_street: String?,
```

Comment:
We can avoid using the property names in snake case by using the @serializedName annotation

Reply:
@vinybazterrica please apply the changes